cli/src/main/java/org/apache/uniffle/cli/ExampleCLI.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.uniffle.cli;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.uniffle.AbstractCustomCommandLine;
+import org.apache.uniffle.UniffleCliArgsException;
+
+
+public class ExampleCLI extends AbstractCustomCommandLine {
```

Comment:
Could we have a better name?

Reply:
What about `UniffleTestCLI`?

Reply:
Why do we need an ExampleCli? Can we have a UniffleCli directly?

Reply:
> Why do we need an ExampleCli? Coulw we have a UniffleCli directly?

Thank you for your suggestion! I will modify the class name, and I will continue to complete subTask in this class, #837, #846.

My idea is to provide a sample class for testing, so it is called `exampleCLI`, but it is called `uniffleCLI`, which is a better choice.